delta/main.py

```diff
@@ -105,8 +108,21 @@ def main(argv):
     raise ValueError("Not support command: {}.".format(FLAGS.cmd))
 
 
-if __name__ == '__main__':
+def entry():
   define_flags()
+  flags.DEFINE_bool('only_nlp', 'False', 'only use nlp modules')
   logging.info("Deep Language Technology Platform start...")
   app.run(main)
   logging.info("OK. Done!")
+
+
+def nlp_entry():
```

Comment:
same entry and nlp_entry

Reply:
The `nlp_entry` is for pip `entry_point`, which set `only_nlp = True` by default.